Practice2/src/common-component.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles/main.css">
+  <title>Common Component</title>
+</head>
+
+<body>
+  <div>
+    <h4>Buttons</h4>
+    <button type="button" class="btn btn-primary">Purchase</button>
+    <button type="button" class="btn btn-secondary">Purchase</button>
+  </div>
+  <div>
+    <h4>Cards</h4>
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title card-title-primary">Silver</h5>
+        <h4 class="card-text-number card-text-number-primary">$49</h4>
+        <h5 class="card-title">Year</h5>
+        <p class="card-text card-text-primary">Personal account ....</p>
+        <a href="javascript:void(0)" class="btn btn-primary">Purchase</a>
+      </div>
+    </div>
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title card-title-secondary">SILVER</h5>
```

Comment:
@duylong13 Seem like you missed this one.

Reply:
Fixed